tests/Unit/API/Site/Controllers/Ads/AdsSettingsControllerTest.php

```diff
@@ -49,6 +49,20 @@ public function test_get_settings() {
 		$this->assertEquals( 200, $response->get_status() );
 	}
 
+	public function test_get_settings_with_null_option_value() {
+		$expected = [
+			'enhanced_conversions_enabled' => false,
+		];
+
+		$this->options->expects( $this->once() )->method( 'get_ads_id' )->willReturn( 1 );
+		$this->options->expects( $this->once() )->method( 'get' )->willReturn( null );
+
+		$response = $this->do_request( self::ROUTE_SETTINGS, 'GET' );
+
+		$this->assertEquals( $expected, $response->get_data() );
+		$this->assertEquals( 200, $response->get_status() );
+	}
+
 	public function test_edit_settings() {
```

Comment:
```suggestion
	public function test_edit_settings() {
```

Duplicate of `test_edit_settings_with_null_option_value` test.

Reply:
@mukeshpanchal27 I don't think this is a duplicate as the new test added is for the GET request, where as the `test_edit_settings_with_null_option_value` is for the POST request.

The new test is to ensure that we return the correct default if the option is not saved in the database. Let me know if you think differently here.

Thanks

Reply:
In that case can we use dataProvide and merge both tests?